Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -103,6 +103,20 @@ python3 create-external-cluster-resources.py --upgrade --rbd-data-pool-name repl
     An existing non-restricted user cannot be converted to a restricted user by upgrading.
     The upgrade flag should only be used to append new permissions to users. It shouldn't be used for changing a csi user already applied permissions. For example, you shouldn't change the pool(s) a user has access to.
 
+### Admin privileges
+
+If in case the cluster needs the admin keyring to configure, here are the steps to use them.
```

Comment:
```suggestion
If in case the cluster needs the admin keyring to configure, Update all the caps to * for the `client.healthchecker`
```

Since this as only one step let' keep it one line